src/TestRotations.java

```diff
@@ -0,0 +1,55 @@
+//: object/TestRotations.java
+
+import junit.framework.*;
+
+/**
+ * JUnit Class to test the rotation methods from the Quaternion and RotationMatrix classes.
+ *
```

Comment:
Maybe instead of having this what you should do is testing the rotation in quaternion with a matrix and the other way around :)